app/static/js/events/wizard/vue-app.js

```diff
@@ -171,6 +177,7 @@ var app = new Vue({
             ticket.sales_start_time = moment().tz(this.event.timezone).format('HH:mm');
             ticket.sales_end_date = moment().tz(this.event.timezone).add(10, 'days').format('MM/DD/YYYY'),
             ticket.sales_end_time = moment().tz(this.event.timezone).add(10, 'days').hour(22).minute(0).format('HH:mm'),
+            ticket.price = 0.01,
```

Comment:
@niranjan94 there are commas on the above two lines as well , they were their originally too, Should I change them too?